supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/api/PipelinesApiRequestWorker.scala

```diff
@@ -56,7 +56,15 @@ class PipelinesApiRequestWorker(val pollingManager: ActorRef, val batchInterval:
   
   // These are separate functions so that the tests can hook in and replace the JES-side stuff
   private[api] def createBatch(): BatchRequest = batch
-  private[api] def runBatch(batch: BatchRequest): Unit = if (batch.size() > 0) batch.execute()
+  private[api] def runBatch(batch: BatchRequest): Unit = {
+    try {
+      if (batch.size() > 0) batch.execute()
+    } catch {
+      case e: java.io.IOException =>
+        val msg = s"A batch of PAPI status requests failed. This is a recoverable error, the request manager will retry automatically. The error was: ${e.getMessage}"
```

Comment:
This comment is intended to be a reflection of the existing code, not a new assertion; it seems that we retry indefinitely even for non-recoverable errors.

Reply:
I believe the retries are capped at 10, but that could be another request type

Reply:
Ah yes, it does say right there
```
private val maxRetries = 10
```